example/create_cloudsql_connection/main.tf

```diff
@@ -0,0 +1,129 @@
+resource "random_id" "random_id" {
+  byte_length = 8
+}
+
+// create a google service account to become dataset owner
+resource "google_service_account" "owner" {
+  account_id   = "datasetowner${random_id.random_id.hex}"
+  display_name = "dataset_owner"
+}
+
+// creating the BigQuery dataset at Jakarta
+module "create_bigquery_dataset" {
```

Comment:
Should drop the `create_` prefixes from resource and module names.

Terraform code is declarative, resources and modules refer to something.  Not all operations in terraform are creating the resource, eg `terraform destroy -target module.bigquery_dataset`, is very weird with the prefix.

Reply:
Thank you Martyn. I just renamed it.